src/schema/metadata/AssociatedEmptyRoom.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: AssociatedEmptyRoom
+description: |
+  Relative path in BIDS folder structure to empty-room file associated with
+  the subject’s MEG recording.
+  The path needs to use forward slashes instead of backward slashes
+  (for example,
+  "sub-emptyroom/ses-/meg/sub-emptyroom_ses-_task-noise_run-_meg.ds").
+anyOf:
```

Comment:
In the specification table, this is a string. In the validator, it's either a string or an array of strings.

Reply:
that's not good :confused:  the validator was accepting arrays for 3 years of more, according to the [git blame](https://github.com/bids-standard/bids-validator/blame/89bf34a68da8f96b4051a4be04e3919f14a77931/bids-validator/validators/json/schemas/meg.json#L48) ... I am not sure whether

1. to adjust the validator not not accept arrays anymore
2. to adjust the spec to also accept arrays in addition to just strings

I am tending towards 2.

Reply:
I feel like 2 couldn't hurt, even if `AssociatedEmptyRoom` won't, in practice, end up being an array. But I know so little about MEG that I have no clue if an array is possible.

Reply:
You can think of an empty room recording as a recording using the MEG machine ... but without a human participant --> the resulting data is then used to model a "noise covariance matrix".

Often these recordings are done regularly ... and for analyzing actual human data, you may want to use an empty-room recording that is as close as possible to the human data recording date in time.

So yes, it's technically possible to ship multiple empty room recordings ... not sure how it's used in practice. It's also minor, so I don't wanna spam people with this, let's do 2.

Reply:
In that case, I will leave this definition as-is and resolve this thread. Thanks!